CMakeLists.txt

```diff
@@ -507,6 +433,99 @@ if(NRN_ENABLE_CORENEURON)
   endif()
 endif()
 
+# =============================================================================
+# Collect the environment variables that are needed to execute NEURON from the build directory. This
+# is used when configuring tests, and when building documentation targets. TODO: be more careful
+# about trailing colons?
+# =============================================================================
+set(NRN_RUN_FROM_BUILD_DIR_ENV
+    "LD_LIBRARY_PATH=${PROJECT_BINARY_DIR}/lib:$ENV{LD_LIBRARY_PATH}"
+    "NEURONHOME=${PROJECT_BINARY_DIR}/share/nrn" "NRNHOME=${PROJECT_BINARY_DIR}"
+    "PATH=${PROJECT_BINARY_DIR}/bin:$ENV{PATH}")
+if(NRN_ENABLE_CORENEURON AND NOT coreneuron_FOUND)
+  # CoreNEURON is enabled *and* is being built internally
+  list(APPEND NRN_RUN_FROM_BUILD_DIR_ENV "CORENRNHOME=${PROJECT_BINARY_DIR}")
+endif()
+if(NRN_ENABLE_PYTHON)
+  list(APPEND NRN_RUN_FROM_BUILD_DIR_ENV
+       PYTHONPATH=${PROJECT_BINARY_DIR}/lib/python:${PROJECT_SOURCE_DIR}/test/rxd:$ENV{PYTHONPATH})
+endif()
+
+if(NRN_ENABLE_DOCS)
+  # Make sure all dependencies are available
+  find_package(Doxygen REQUIRED)
```

Comment:
We also need `ffmpeg` and `pandoc`

Reply:
Added. CMake does not ship modules to find them, so I just check that the `ffmpeg` and `pandoc` executables can be found.